Tasks/Common/webdeployment-common-v2/jsonvariablesubstitutionutility.ts

```diff
@@ -73,19 +73,19 @@ export function substituteJsonVariableV2(jsonObject, envObject) {
             if(resultNode.isEnd) {
                 switch(typeof(jsonObject[jsonChild])) {
                     case 'number':
-                    tl.debug('substituting value on key: ' + jsonChild + ' with (number) value: ' + resultNode.value);
+                        console.log(tl.loc('SubstitutingValueonKeyWithNumber' + jsonChild + resultNode.value));
```

Comment:
While using tl.loc (for locatized strings), instead of using + to append to string, use a placeholder eg.
tl.loc('SubstitutingValueKeyWithNumber', jsonChild,resultnode.value)
While defining string in task.json say:
"SubstitutingValueonKeyWithNumber" : "Substituting value on key: %s with (number) value: %s"

Do this for all loc strings